src/feature.py

```diff
@@ -0,0 +1,169 @@
+from chem_lex.ChemlabTokens import unit_prefix, supported_units
+#import
+from element import Element
+def convertTo(el, num, Unit, newUnit):
+    ### need to redo with the parser
+
+    if(Unit in unit_prefix and Unit in unit_prefix):
```

Comment:
It's comparing the same thing twice? Think something may be wrong here. Clean up redundancy